lib/logger.js

```diff
@@ -140,7 +140,9 @@ function Logger(key, options) {
     this._req = {
         auth: { username: key }
         , agent: useHttps ? new Agent.HttpsAgent(configs.AGENT_SETTING) : new Agent(configs.AGENT_SETTING)
-        , headers: clone(configs.DEFAULT_REQUEST_HEADER)
+        , headers: Object.assign({}, clone(configs.DEFAULT_REQUEST_HEADER), {
```

Comment:
Why do you need the `{}` here? Won't `clone(configs.DEFAULT_REQUEST_HEADER)` give you something acceptable?

Reply:
It's for making sure of nothing crazy is gonna happen in this merge of 2 objects - it's there just for safety reasons - no harm it has with its existence